### ISSUE TITLE
[axe-core warning] - Ensure <li> elements are used semantically

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-framework#654]
# Feature Request

Ensure `<li>` elements are used semantically. `<li>` elements must be contained in a `<ul>` or `<ol>`.

## Description
<details>
 <summary>WDIO Axe Error Report</summary>

```
Collapsible Menu View Responsive Hidden Open is accessible:
expected no accessibility violations but got:
        {
  "description": "Ensures <li> elements are used semantically",
  "help": "<li> elements must be contained in a <ul> or <ol>",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/listitem?application=axeAPI",
  "id": "listitem",
  "impact": "serious",
  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": null,
          "id": "listitem",
          "impact": "serious",
          "message": "List item does not have a <ul>, <ol> or role=\"list\" parent element",
          "relatedNodes": []
        }
      ],
      "failureSummary": "Fix any of the following:\n  List item does not have a <ul>, <ol> or role=\"list\" parent element",
      "html": "<li>",
      "impact": "serious",
      "none": [],
      "target": [
        ".Menu-module__list___2zyhl > li:nth-child(3)"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": null,
          "id": "listitem",
          "impact": "serious",
          "message": "List item does not have a <ul>, <ol> or role=\"list\" parent element",
          "relatedNodes": []
        }
      ],
      "failureSummary": "Fix any of the following:\n  List item does not have a <ul>, <ol> or role=\"list\" parent element",
      "html": "<li>",
      "impact": "serious",
      "none": [],
      "target": [
        "li:nth-child(6)"
      ]
    }
  ],
  "tags": [
    "cat.structure",
    "wcag2a",
    "wcag131"
  ]
}
running chrome
AssertionError: expected no accessibility violations but got:
        {
  "description": "Ensures <li> elements are used semantically",
  "help": "<li> elements must be contained in a <ul> or <ol>",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/listitem?application=axeAPI",
  "id": "listitem",
  "impact": "serious",
  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": null,
          "id": "listitem",
          "impact": "serious",
          "message": "List item does not have a <ul>, <ol> or role=\"list\" parent element",
          "relatedNodes": []
        }
      ],
      "failureSummary": "Fix any of the following:\n  List item does not have a <ul>, <ol> or role=\"list\" parent element",
      "html": "<li>",
      "impact": "serious",
      "none": [],
      "target": [
        ".Menu-module__list___2zyhl > li:nth-child(3)"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": null,
          "id": "listitem",
          "impact": "serious",
          "message": "List item does not have a <ul>, <ol> or role=\"list\" parent element",
          "relatedNodes": []
        }
      ],
      "failureSummary": "Fix any of the following:\n  List item does not have a <ul>, <ol> or role=\"list\" parent element",
      "html": "<li>",
      "impact": "serious",
      "none": [],
      "target": [
        "li:nth-child(6)"
      ]
    }
  ],
  "tags": [
    "cat.structure",
    "wcag2a",
    "wcag131"
  ]
}
    at Context.<anonymous> (/Users/bj031910/repos/terra-framework/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/accessiblity.js:13:54)
    at new Promise (<anonymous>)
    at new F (/Users/bj031910/repos/terra-framework/node_modules/core-js/library/modules/_export.js:36:28)
```
</details>

<details>
 <summary>WDIO Axe Error Report</summary>

```
Collapsible Menu View Responsive Hidden Open is accessible:
expected no accessibility violations but got:
        {
  "description": "Ensures <li> elements are used semantically",
  "help": "<li> elements must be contained in a <ul> or <ol>",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/listitem?application=axeAPI",
  "id": "listitem",
  "impact": "serious",
  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": null,
          "id": "listitem",
          "impact": "serious",
          "message": "List item does not have a <ul>, <ol> or role=\"list\" parent element",
          "relatedNodes": []
        }
      ],
      "failureSummary": "Fix any of the following:\n  List item does not have a <ul>, <ol> or role=\"list\" parent element",
      "html": "<li>",
      "impact": "serious",
      "none": [],
      "target": [
        ".Menu-module__list___2zyhl > li:nth-child(3)"
      ]
    }
  ],
  "tags": [
    "cat.structure",
    "wcag2a",
    "wcag131"
  ]
}
running chrome
AssertionError: expected no accessibility violations but got:
        {
  "description": "Ensures <li> elements are used semantically",
  "help": "<li> elements must be contained in a <ul> or <ol>",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/listitem?application=axeAPI",
  "id": "listitem",
  "impact": "serious",
  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": null,
          "id": "listitem",
          "impact": "serious",
          "message": "List item does not have a <ul>, <ol> or role=\"list\" parent element",
          "relatedNodes": []
        }
      ],
      "failureSummary": "Fix any of the following:\n  List item does not have a <ul>, <ol> or role=\"list\" parent element",
      "html": "<li>",
      "impact": "serious",
      "none": [],
      "target": [
        ".Menu-module__list___2zyhl > li:nth-child(3)"
      ]
    }
  ],
  "tags": [
    "cat.structure",
    "wcag2a",
    "wcag131"
  ]
}
    at Context.<anonymous> (/Users/bj031910/repos/terra-framework/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/accessiblity.js:13:54)
    at new Promise (<anonymous>)
    at new F (/Users/bj031910/repos/terra-framework/node_modules/core-js/library/modules/_export.js:36:28)
```
</details>
